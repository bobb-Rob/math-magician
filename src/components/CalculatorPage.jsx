import React from 'react';
import Calculator from './calculator';

const CalculatorPage = () => (
  <div>
    CalculatorPage
    <Calculator />
  </div>
);

export default CalculatorPage;
