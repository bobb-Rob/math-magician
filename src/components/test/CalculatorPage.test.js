import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorPage from '../CalculatorPage';

test('test id Calculator page is rendered', () => {
  const CalcPage = renderer.create(<CalculatorPage />).toJSON();
  expect(CalcPage).toMatchSnapshot();
})