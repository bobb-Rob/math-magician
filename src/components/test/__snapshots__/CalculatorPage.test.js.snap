// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test id Calculator page is rendered 1`] = `
<div
  className="calc-page"
>
  <h3>
    Lets do some math!
  </h3>
  <div
    className="calculator-container"
  >
    <div
      className="cal-screen"
    >
      <span />
    </div>
    <form
      className="calculator-digits"
      name="calculator"
    >
      <input
        name="AC"
        onClick={[Function]}
        type="button"
        value="AC"
      />
      <input
        name="+/-"
        onClick={[Function]}
        type="button"
        value="+/-"
      />
      <input
        name="%"
        onClick={[Function]}
        type="button"
        value="%"
      />
      <input
        className="peach-color divide"
        name="÷"
        onClick={[Function]}
        type="button"
        value="÷"
      />
      <input
        name="7"
        onClick={[Function]}
        type="button"
        value="7"
      />
      <input
        name="8"
        onClick={[Function]}
        type="button"
        value="8"
      />
      <input
        name="9"
        onClick={[Function]}
        type="button"
        value="9"
      />
      <input
        className="peach-color"
        name="x"
        onClick={[Function]}
        type="button"
        value="×"
      />
      <input
        name="4"
        onClick={[Function]}
        type="button"
        value="4"
      />
      <input
        name="5"
        onClick={[Function]}
        type="button"
        value="5"
      />
      <input
        name="6"
        onClick={[Function]}
        type="button"
        value="6"
      />
      <input
        className="peach-color"
        name="-"
        onClick={[Function]}
        type="button"
        value="−"
      />
      <input
        name="3"
        onClick={[Function]}
        type="button"
        value="3"
      />
      <input
        name="2"
        onClick={[Function]}
        type="button"
        value="2"
      />
      <input
        name="1"
        onClick={[Function]}
        type="button"
        value="1"
      />
      <input
        className="peach-color"
        name="+"
        onClick={[Function]}
        type="button"
        value="ᐩ"
      />
      <input
        className="zero"
        name="0"
        onClick={[Function]}
        type="button"
        value="0"
      />
      <input
        name="."
        onClick={[Function]}
        type="button"
        value="."
      />
      <input
        className="peach-color"
        name="="
        onClick={[Function]}
        type="button"
        value="="
      />
    </form>
  </div>
</div>
`;
